pontoon/base/static/js/fluent_interface.js

```diff
@@ -23,6 +23,57 @@ var Pontoon = (function (my) {
       );
   }
 
+  function renderOriginalVariant(value, elements) {
+    return '<li>' +
+      '<span class="id">' +
+        value +
+      '</span>' +
+      '<span class="value">' +
+        Pontoon.fluent.serializePlaceables(elements, true) +
+      '</span>' +
+    '</li>';
+  }
+
+  function renderOriginalSimpleElement(simpleElements, title) {
+    if (simpleElements.length) {
+      var response = renderOriginalVariant(title, simpleElements);
+      simpleElements = [];
+      return response;
+    }
+
+    return '';
+  }
+
+  /*
+   * Render original string elements.
+   */
+  function renderOriginalElements(elements, title) {
+    var content = '';
+    var simpleElements = [];
+
+    elements.forEach(function (element, i, array) {
```

Comment:
`array` is a poor variable name as it is really close to `Array`. I would favor a different name here.